Dockerfile-dev

```diff
@@ -12,6 +12,9 @@ ENV BIGCHAINDB_SERVER_BIND 0.0.0.0:9984
 ENV BIGCHAINDB_WSSERVER_HOST 0.0.0.0
 ENV BIGCHAINDB_WSSERVER_SCHEME ws
 
+ENV BIGCHAINDB_WSSERVER_ADVERTISED_HOST 0.0.0.0
+ENV BIGCHAINDB_WSSERVER_ADVERTISED_SCHEME ws
+
```

Comment:
Because there is not `BIGCHAINDB_WSSERVER_PORT` in the Dockerfile-dev.
I believe we have #1704  which will help @sbellem and me to answer what can and cannot get included in the Dockerfiles. I guess we can follow that one to resolve the case with the Dockerfiles.